resources/assets/coffee/react/profile-page/extra.coffee

```diff
@@ -19,21 +19,57 @@
 el = React.createElement
 
 class ProfilePage.Extra extends React.Component
+  # Mirrored from App/Models/User.php
+  # The numbers corresponding to particular profile section.
+  USER_PAGE = 1
```

Comment:
Additionally, if it's for db storage saving, the integer can be converted immediately when storing/retrieving from db during casting phase so nothing else will need to do the mapping. If it's for bandwidth saving, then, well ┐('～`；)┌ (not sure if worth the effort)